docs/library/forms/editor.md

```diff
@@ -4,144 +4,3 @@ components:
 ---
 
 # Editor
-
-An HTML editor component based on [Suneditor](http://suneditor.com/sample/index.html).
```

Comment:
why is this doc removed?

Reply:
Was causing some issues, trying to bring it back